### ISSUE TITLE
Build pot (po and mo) also for pages and articles

### ISSUE THREAD

**Issue by ghtmtt** [getpelican/pelican-plugins#878]
Hi,

hoping this is the right place where to ask this question. 
I started to build a pelican website (first time I'm using pelican). I found this plugin really super useful for me.

My website is in both en and it (for now). I'm using [this guide](https://github.com/getpelican/pelican-plugins/blob/master/i18n_subsites/localizing_using_jinja2.rst) to translate all the templates (using bootstrap theme) and all is going straight.

Moreover, my *strategy* to have pages and articles in different languages is to have a **content** tree like this:

```
├── blog
│   ├── en
│   │   ├── post.md
│   └── it
│       └── post_it.md
├── extras
├── images
└── pages
    ├── en
    │   ├── about.md
    │   ├── blog.md
    │   ├── courses.md
    └── it
        ├── about_it.md
        ├── blog_it.md
        ├── courses_it.md
```
where all article and pages have the metadata **lang**.

My question now is: is it possible in some way to avoid thee duplication of articles and pages and use the template translation strategy?

So in other words, have an unique folder with all the article and pages with the **lang** metadata (e.g. set to en that is the default language) and then having an additional metadata **t_lang** with all the translation languages?

Thanks for any feedback and thanks for the great plugin!

**Comment by justinmayer** [comment]
@smartass101: Any thoughts on this?

**Comment by justinmayer** [comment]
Closing due to inactivity. Please feel free to comment here if you feel the issue should be re-opened.
